hostinit/setup_test.go

```diff
@@ -1,310 +0,0 @@
-package hostinit
```

Comment:
I wonder/worry that we've lost some test coverage by removing this, because the refactored code in the units package may not be as well tested.

Reply:
fixed